### PR TITLE
hardware base documentations

### PR DESCRIPTION
**Pull request by anindex**
The PRs proposes documentations for these components:
- differentialrobotComp
- delfos
- robexComp

### CODE REVIEW

File: components/hardware/base/delfos/README.md [hunk 1]
```diff
@@ -1,38 +1,60 @@
+# delfos
+
+`delfos` component is implemented as generic controller for omnidirectional robot (e.g omni wheels robot) in RoboComp left-handed coordinate. The follow list specifies set of implemented functions for controlling and robot state measuring as specified in interface `OmniRobot.idsl`:
```
Comment (orensbruli) [inline components/hardware/base/delfos/README.md]: I think it's important to talk about the Left-handed coordinate system that Luis Vicente linked on his document:
https://www.evl.uic.edu/ralph/508S98/coordinates.htm
Not just mentionit  but explain in a paragraph how it works and the expected differences. It usually brings so much confusion when new people try to work with these components.

File: components/hardware/base/delfos/README.md [hunk 2]
```diff
@@ -1,38 +1,64 @@
+# delfos
+
+`delfos` component is implemented as generic controller for omnidirectional robot (e.g omni wheels robot) in RoboComp left-handed coordinate. The left-handed coordinate has the positive x, y and z axes point right, up and forward, respectively (see picture below. Credit: https://www.oreilly.com). Positive rotation is clockwise about the axis of rotation.
+
+![coordinate](https://www.oreilly.com/library/view/learn-arcore-/9781788830409/assets/a465e4c5-b6ca-4006-a40e-1aa9ad2ebc5d.png)
+
+For developer notices, the follow list specifies set of implemented functions for controlling and robot state measuring as specified in interface `OmniRobot.idsl`:
```
Comment (pilarbachiller) [inline components/hardware/base/delfos/README.md]:
```suggestion
For developer notices, the following list specifies the set of implemented functions for controlling and robot state measuring as specified in interface `OmniRobot.idsl`:
```

File: components/hardware/base/delfos/README.md [hunk 3]
```diff
@@ -1,38 +1,64 @@
...
+
+- **getBaseState(RoboCompGenericBase::TBaseState  &state)**: gets robot state type `TBaseState`.
```
Comment (pilarbachiller) [inline components/hardware/base/delfos/README.md]:
```suggestion
- **getBaseState(RoboCompGenericBase::TBaseState  &state)**: gets the robot state (type `TBaseState`).
```

File: components/hardware/base/differentialrobotComp/README.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
+# DifferentialRobotComp
+
+`DifferentialRobotComp` component is implemented as generic controller for differential robot (e.g two wheels robot) in RoboComp left-handed coordinate. For developer notices,  the follow list specifies set of implemented control functions as specified in interface `DifferentialRobot.ice`:
```
Comment (pilarbachiller) [inline components/hardware/base/differentialrobotComp/README.md]:
```suggestion
`DifferentialRobotComp` component is implemented as generic controller for differential robot (e.g two wheels robot) in RoboComp left-handed coordinate. For developer notices,  the following list specifies the set of implemented control functions as specified in interface `DifferentialRobot.ice`:
```

File: components/hardware/base/differentialrobotComp/README.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+
+- **getBaseState(RoboCompGenericBase::TBaseState  &state)**: gets robot state type `TBaseState`.
```
Comment (pilarbachiller) [inline components/hardware/base/differentialrobotComp/README.md]:
```suggestion
- **getBaseState(RoboCompGenericBase::TBaseState  &state)**: gets the robot state (type `TBaseState`).
```

File: components/hardware/base/robexComp/README.md [hunk 6]
```diff
@@ -1,36 +1,80 @@
+# RobexComp
+
+`RobexComp` component is a controller software for custom-made mobile robot in Robolab, University of Extremadura. The controller component consists of `JoystickAdapter.idsl` for sending control data message, `DifferentialRobot.idsl` for reading and controlling specific dynamics of differential robot and `GenericBase.idsl` for reading general robot states. For developer notices, the follow list specifies set of implemented functions for each of the interfaces:
```
Comment (pilarbachiller) [inline components/hardware/base/robexComp/README.md]:
```suggestion
`RobexComp` component is a controller software for custom-made mobile robot in Robolab, University of Extremadura. The controller component consists of `JoystickAdapter.idsl` for sending control data message, `DifferentialRobot.idsl` for reading and controlling specific dynamics of differential robot and `GenericBase.idsl` for reading general robot states. For developer notices, the following list specifies the set of implemented functions for each of the interfaces:
```

**Pull request merged**